README.md

```diff
@@ -1,37 +1,71 @@
-# Augmented Reality Based Resume With FaceRecognition
-An augmented reality based resume with Face recognition. The iOS app recognizes the face and presents you with the AR view that contains 3D mock face and details of your resume.
+# Augmented Reality based Résumé with Visual Recognition
+In this code pattern, we will create augmented reality based résumé with Visual recognition. The iOS app recognizes the face and presents you with the AR view that contains details of the person in the camera view. The app utilizes IBM Visual Recognition to classify the image and uses that classification to get details about the person that is stored in IBM Cloudant
 
 
-# Architecture
+# Flow
+
+![ARResume Architecture](https://github.com/IBM/ar-resume-with-visual-recognition/blob/readme.update/ARResume.png "ARResume Architecture")
```

Comment:
you can write this as:

`![ARResume Architecture](ARResume.png)`

and 

`![](ARResume.png)`